jwst/master_background/master_background_step.py

```diff
@@ -252,12 +253,11 @@ def split_container(container):
         background.append(container._models[ind_bkgd])
 
     # Pass along the association table to the output science container
-    science.meta.asn_table = {}
     science.asn_pool_name = container.asn_pool_name
     science.asn_table_name = container.asn_table_name
-    merge_tree(science.meta.asn_table.instance, asn)
+    merge_tree(science.asn_table, container.asn_table)
     # Prune the background members from the table
-    for p in science.meta.asn_table.instance['products']:
+    for p in science.asn_table['products']:
```

Comment:
Current default values for the asn_table might do bad things here too - maybe make members an empty list?

Reply:
will test this